From 586543c75320453ed0a8cf422b68f694b337a1bb Mon Sep 17 00:00:00 2001
From: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date: Fri, 1 Jan 2021 19:01:08 +0100
Subject: [PATCH 26/98] dt-bindings: clock: meson8b: add the rtc_32k oscillator
 input

The CLK81 tree can be driven off the 32kHz oscillator connected to the
SoCs RTC32K_XI and RTC32K_XO pads. Add this clock as a valid input.

Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
---
 .../devicetree/bindings/clock/amlogic,meson8-clkc.yaml         | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/clock/amlogic,meson8-clkc.yaml b/Documentation/devicetree/bindings/clock/amlogic,meson8-clkc.yaml
index 25e25c4dd0b806..f3b11afebbca7d 100644
--- a/Documentation/devicetree/bindings/clock/amlogic,meson8-clkc.yaml
+++ b/Documentation/devicetree/bindings/clock/amlogic,meson8-clkc.yaml
@@ -21,7 +21,7 @@ properties:
 
   clocks:
     minItems: 2
-    maxItems: 4
+    maxItems: 5
 
   clock-names:
     minItems: 2
@@ -30,6 +30,7 @@ properties:
       - const: ddr_pll
       - const: clk_32k
       - const: rmii_clk
+      - const: rtc_32k
 
   '#clock-cells':
     const: 1

